docs/release-notes/8.16.asciidoc

```diff
@@ -1,6 +1,41 @@
 [[release-notes-header-8.16.0]]
 == 8.16
 
+[discrete]
+[[release-notes-8.16.5]]
+=== 8.16.5
+
+[discrete]
+[[known-issue-8.16.5]]
+==== Known issues
+
+// tag::known-issue[]
+[discrete]
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
+
+[discrete]
+[[bug-fixes-8.16.5]]
+==== Bug fixes
+* Fixes a bug in {elastic-defend} for Linux where tty capture limit defaults were ignored.
+* Fixes a potential {elastic-defend} crash when generating multiple ransomware alerts on Windows. This issue was simultaneously mitigated by a cloud artifact update (manifest version 1.0.1381) on February 24, 2025. Internet-connected instances of {elastic-defend} will automatically receive this update -- no user intervention required. Air-gapped customers hosting their own artifacts should follow [these instructions](offline-endpoint). We would like to acknowledge [Todyl](https://www.todyl.com) for their assistance with this issue.
```

Comment:
Broken links

Reply:
Fixed in [eb3cf87](https://github.com/elastic/security-docs/pull/6551/commits/eb3cf877fe137bba1631c0b186a19c5a979e0ade).